assets/src/edit-story/app/layout/index.js

```diff
@@ -52,7 +52,6 @@ const Editor = styled.div`
 const Area = styled.div`
   grid-area: ${({ area }) => area};
   position: relative;
-  overflow: hidden;
```

Comment:
Required to show color picker next to inspector. Is it an issue?

Reply:
By itself, no. I introduced it to ensure that we don't forget to make sections scrollable, otherwise they break the main layout. Possible less of an issue now since most of those sections are already outlined.

Reply:
To reiterate - i think it's fine to remove this. But, as an alternative, maybe it's better to portal the color picker and other popups like that onto the document's body or story's top-level root? Otherwise, this seems rather fragile - there could always be a need to enter an extra layer with overflow hidden.

Reply:
I've added a more general sidebar component to the top level in dbba2dee.